### ISSUE TITLE
2.7.0: ssh has no method runInOrder

### ISSUE THREAD

**Issue by mvysny** [int128/gradle-ssh-plugin#287]
Hi, I have a very simple SSH script as follows:
```groovy
task deploy(description: 'Deploys to the production', group: 'deploy') {
  doLast {
    ssh.runInOrder {
    }
  }
}
```

Yet it fails with:
```
* What went wrong:
Execution failed for task ':deploy'.
> No signature of method: org.hidetake.groovy.ssh.core.Service.runInOrder() is applicable for argument types: (build_dloae872cdcjb1uwzaxi52fkn$_run_closure6$_closure15$_closure16) values: [build_dloae872cdcjb1uwzaxi52fkn$_run_closure6$_closure15$_closure16@633d04bd]
```

### Environment info
Please paste the output of `println ssh.version` e.g. `gradle-ssh-plugin-x.x.x (groovy-ssh-x.x.x, jsch-0.1.x, groovy-2.4.4, java-1.7.0_101)`.

If it is difficult, describe version of the plugin, JVM and OS.

```
gradle-ssh-plugin-2.7.0 (groovy-ssh-2.7.0, jsch-0.1.53, groovy-2.4.7, java-1.8.0_111)
```

### Steps to reproduce
1. Create the `build.gradle` file as below
2. Run gradle deploy
3. Gradle 3.3 fails with an exception

```groovy
plugins {
    id 'org.hidetake.ssh' version '2.7.0'
}

task deploy(description: 'Deploys to the production', group: 'deploy') {
  doLast {
    ssh.runInOrder {
    }
  }
}
```

**Comment by int128** [comment]
`runInOrder` is available since 2.7.1. Please try the new release.

**Comment by mvysny** [comment]
Thank you, it works perfectly.
